.concourse/tasks/upgrade.sh

```diff
@@ -15,16 +15,19 @@ export FORCE_DELETE=true
 export HELM_VERSION="v3.1.1"
 export BACKEND=imported
 export QUIET_OUTPUT=true
-export GKE_CLUSTER_NAME="kubecf-ci-$(cat semver.gke-cluster/version | sed 's/\./-/g')"
-export KUBECFG="$(readlink -f ~/.kube/config)"
+GKE_CLUSTER_NAME="kubecf-ci-$(sed 's/\./-/g' < "semver.gke-cluster/version")"
+export GKE_CLUSTER_NAME
+KUBECFG="$(readlink -f ~/.kube/config)"
```

Comment:
```suggestion
KUBECFG="$(readlink --canonicalize ~/.kube/config)"
```
`readlink -f` is a GNU-specific option anyway (Darwin/BSD readlink has `-f` as _Display information using the specified format_), so using the long option doesn't lose us anything.